src/Features/VisualBasic/Portable/UseInferredMemberName/VisualBasicUseInferredMemberNameCodeFixProvider.vb

```diff
@@ -33,7 +33,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseInferredMemberName
                 Dim node = root.FindNode(diagnostic.Location.SourceSpan)
                 Select Case node.Kind
                     Case SyntaxKind.NameColonEquals
-                        editor.RemoveNode(node, SyntaxRemoveOptions.KeepExteriorTrivia)
+                        editor.RemoveNode(node, SyntaxRemoveOptions.KeepExteriorTrivia Or SyntaxRemoveOptions.AddElasticMarker)
                         Exit Select
 
                     Case SyntaxKind.NamedFieldInitializer
```

Comment:
is there a similar issue for NamedFieldInitializer?

Reply:
Not as far as I can tell. I have spaces in VB object initializers in various positions and also tested a couple more, but I did not notice any misbehavior.